lib/hexpm_web/templates/user/show.html.eex

```diff
@@ -1,37 +1,141 @@
-<h2><%= @user.username %></h2>
-
-<div class="row">
-  <div class="col-sm-9">
-    <%= if @user.owned_packages == [] do %>
-      <h3>No owned packages</h3>
-    <% else %>
-      <div class="with-divider user-package-info clearfix">
-        <h3>Total Owned Packages<br /><em><%= length(@user.owned_packages) %></em></h3>
-        <h3>Total Downloads<br /><em><%= ViewHelpers.human_number_space(@total_downloads) %></em></h3>
+<div class="relative">
+  <div class="max-w-7xl mx-auto">
+    <div class="flex flex-row gap-8 px-8">
+      <div class="flex flex-col items-center justify-center w-4/12 h-96 pt-10">
+        <div>
+          <img src="<%= ViewHelpers.gravatar_url(@gravatar_email, :large) %>" class="h-28 w-28 rounded-full">
+        </div>
+        <div class="pt-6">
+          <h4 class="font-semibold text-lg text-gray-900"><%= @user.full_name %></h4>
+        </div>
+        <div class="text-sm text-gray-500 font-normal">
+          <%= if @public_email do %>
+            <a href="mailto:<%= @public_email %>"><%= @public_email %></a>
+          <% end %>
+        </div>
+        <div class="flex flex-row gap-4 py-6">
+          <%= for {icon, url} <- UserHandles.render(@user) do %>
+            <%= if icon == :elixirforum or icon == :freenode do%>
+              <a href="<%= url %>"><%= UserHandles.handle_svg_icon(icon) %></a>
+            <% else %>
+              <a href="<%= url %>"><%= icon(:remixicon, :"#{icon}", width: "16px", fill: "rgba(48, 66, 84, 1)")%></a>
+            <% end %>
+          <% end %>
+        </div>
+        <div class="border-t border-gray-200 pt-6">
+          <div class="grid grid-cols-2 flex flex-row gap-4">
```

Comment:
We can't have both grid and flex, they apply the same `display` CSS property.

Reply:
Thank you for catching that , I should have removed the flex property